lib/maze/servlets/command_servlet.rb

```diff
@@ -16,13 +16,15 @@ def do_GET(_request, response)
         response.header['Access-Control-Allow-Origin'] = '*'
 
         commands = Maze::Server.commands
+
         if commands.size_remaining == 0
-          response.body = 'No commands to provide'
-          response.status = 400
+          response.body = '{"action": "noop", "message": "No commands queued"}'
+          response.status = 200
```

Comment:
I realise this change may be contentious and worth a discussion IRL.

Reply:
Will this effect any currently running pipelines, or just something we should be mindful of going forward?

Reply:
Unity pipeline now tested against this and working fine.